### ISSUE TITLE
IDMP-744 - Investigate simplifying potential circularities in the packaging / container specification part of the ontology

### ISSUE THREAD

**Issue by ElisaKendall** [edmcouncil/idmp#607]
There appear to be some significant recursive definitions related to packaging, at least one of which is in the ISO IDMP 11615 specification. Some of the circularity needs to be simplified to address reasoning challenges.

**Comment by tw-osthus** [comment]
please verify the changes against the Qlaira example. The Amlodipine case is too simple.

**Comment by tw-osthus** [comment]
We can merge container with container constituency, but we still have to use constainer constituents, because in the Qlaira example we have 2 yellow tablets, 8 red tablets, 4 brown tablets,.. in the same blister. We need the amount of content qualified by type. In Amlodipine it is just one.

**Comment by ElisaKendall** [comment]
> please verify the changes against the Qlaira example. The Amlodipine case is too simple.

Agreed. The process will still be incremental - not planning to unwind this all at once.

**Comment by ElisaKendall** [comment]
After discussion with BI, it appears that container constituency is actually redundant. Package item has constituent something and something is a constituent of a package item should be retained, with elimination of the need for is defined in related to the constituency. This change will simplify the BI implementation, and likely others, significantly.
